service/service.go

```diff
@@ -176,6 +199,22 @@ func (s *Service) Shutdown() {
 	s.Stop <- os.Signal(os.Interrupt)
 }
 
+func listenAndServeTLS(server *http.Server, certFile, keyFile string, errChan chan error) {
+	if err := server.ListenAndServeTLS(certFile, keyFile); err != nil {
+		errChan <- err
+		close(errChan)
```

Comment:
You probably should not close the channel here. We might have a panic on 213.

Reply:
I don't think so, In `166` you see that this chan is of size of 1, after we send an error, we have a fatal on `183` which will close servers anyway.